tests/claim-project.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { test, expect } from '@playwright/test';
+import { execa } from 'execa';
+
+test('claim project flow', async ({ page }) => {
+  test.setTimeout(240_000);
```

Comment:
Were you going to look at production building to avoid this (assuming this is from dev optimization stuffs)?

Reply:
That is actually necessary regardless, because our tests are necessarily somewhat slow running, just because we need to wait for transactions. This setTimeout defines the timeout for the _entire_ test, which is only 30 seconds by default, which is too little. It'd be necessary even if it ran against a prod image